### DISCUSSION TITLE
物联网与『高效的』IOTA · /iota-tangle

### DISCUSSION THREAD

**Discussion by draveness** [draveness/blog-comments#525] [Announcements]
https://draveness.me/iota-tangle

**Comment by mimirt** [comment]
你好，前陣子看到你的幾篇文章非常欣賞，
不過IOTA正好是我最關注的一個項目，你文中提到的演算法部分我比較不熟悉，
但是關於三進位的硬體，IOTA基金會有一個項目叫做"JINN"就在做這件事，另外IOTA基金會在6月初也公布了一個語言叫做Abra（詳情可見 https://qubic.iota.org ，相信你可能會有興趣）
另外官方的錢包確實難用，不過五、六月以來，新一代的"Trinity"錢包已經進入最終測試階段（iOS及Android版進入beta），桌面版似乎也快了。就我自己的體驗（iOS），是非常簡單、好用的一套錢包。

**Comment by draveness** [comment]
> @mimirt
> 你好，前陣子看到你的幾篇文章非常欣賞，
> 不過IOTA正好是我最關注的一個項目，你文中提到的演算法部分我比較不熟悉，
> 但是關於三進位的硬體，IOTA基金會有一個項目叫做"JINN"就在做這件事，另外IOTA基金會在6月初也公布了一個語言叫做Abra（詳情可見 https://qubic.iota.org ，相信你可能會有興趣）
> 另外官方的錢包確實難用，不過五、六月以來，新一代的"Trinity"錢包已經進入最終測試階段（iOS及Android版進入beta），桌面版似乎也快了。就我自己的體驗（iOS），是非常簡單、好用的一套錢包。

1. 三进制的硬件的研究可能会带来一些性能的提升，但是相比于工程和实现方面的牺牲，这种改变其实也是微不足道的，从目前看也不会有**数量级**的提升；
2. 这门编程语言的完成度目前看来还是挺低的，不知道做出来会有什么革命性的改变；
3. 官方的钱包可以期待一下，现在的真是太难用了

**Comment by sungitly** [comment]
好奇问下楼主的图是用什么工具画的？很漂亮

**Comment by amas-git** [comment]
hello, 实在找不到你的联系方式，发了一封邮件，不知道收到没有
